1.8/tutorial/resources.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: DC/OS 101 - Understanding Ressources
+menu_order: 3
+---
+
+# Prerequisites
+We expect you to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, you have just deployed app2 in our cluster and have verified it is running.
+
+# Objective
+When installing all these apps, we can see that the overall resource utilization in the UI goes up.
+But what does increased resource utilization mean happen, how are resource limits enforced. How can I debug issues arising in the context of resource utilization?
+
+# Steps
+Resource Mangement and resource isolation between task is one of the core challenges of an operating system to manage resources.
+
+  * Deploy App
+    * Let us have a second look at the defintion of [app2](https://github.com/joerg84/dcos-101/blob/master/app2/app2.go).
+  ```
+{
+  "id": "/dcos-101/app2",
+  "cmd": "chmod u+x app2 && ./app2",
+  "args": null,
+  "user": null,
+  "env": null,
+  "instances": 1,
+  "cpus": 1,
+  "mem": 128,
+  "disk": 0,
+  "gpus": 0,
+  ...
+  ```
+  Note this specifies the allocated resources and hence the upper limit of resources which can be used by the task. This is not necessarly the same as acutally used resources which is usually less.
+
+  When needed more resources for our app, we can scale in two dimensions: horizontally and vertically.
```

Comment:
+  When we need more resources for our app, we can scale in two dimensions: horizontally and vertically.